pipeline.py

```diff
@@ -240,13 +252,15 @@ def ilab_pipeline(
     model_source_task = dsl.importer(
         artifact_uri=sdg_base_model, artifact_class=dsl.Model
     )
+    model_source_task.after(prerequisites_check_task)
```

Comment:
After the rebase, you'll need to add `model_tokenizer_source_task.after(prerequisites_check_task)` as well.